setup.py

```diff
@@ -19,7 +19,7 @@
         install_requires = ['setuptools',
             'tiddlyweb>=1.4.2',
             'feedgenerator'],
-        'extras_require': {
+        extras_require = {
```

Comment:
Yeah, this was probably something that never got properly tested and was fixed in haste at some point, badly replicating an on the fly problem. 

Turns out I have this fix in a local checkout and never committed...